steps/yourappbox-app-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,75 @@
+title: YourAppBox App Upload
+summary: |
+  Upload your ipa or apk to YourAppBox.
+description: |
+  Upload your ipa or apk to YourAppBox, also provides legacy uploading to Installfish.
+website: https://github.com/andrewmarmion/bitrise-step-yourappbox-app-upload
+source_code_url: https://github.com/andrewmarmion/bitrise-step-yourappbox-app-upload
+support_url: https://github.com/andrewmarmion/bitrise-step-yourappbox-app-upload/issues
+published_at: 2020-03-21T08:27:04.301907Z
+source:
+  git: https://github.com/andrewmarmion/bitrise-step-yourappbox-app-upload.git
+  commit: 835f883fdf749df53595260591c65ce15f6d1df6
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```

Comment:
No git nor wget seem to be used in script. However, there is a curl, which should be declared here.